src/test/crossFetchNode.test.ts

```diff
@@ -40,6 +40,27 @@ test('test getting a token with a post operation', async () => {
   expect(token).toEqual('Bearer test-auth');
 });
 
+test('test getting a token without a proxy', async () => {
```

Comment:
This is great but would also recommend a negative test when the shortcode isn't set properly and an override test showing you can override the shortcode properly as well.

Reply:
Added more tests. Now we have coverage for valid short code, invalid short code, and missing short code.